cmd/itzo-launcher/itzo-launcher.go

```diff
@@ -36,9 +37,37 @@ var (
 )
 
 var (
-	version = flag.Bool("version", false, "print version and exit")
+	version    = flag.Bool("version", false, "print version and exit")
+	itzoLogDir = flag.String("itzo-log-dir", "/var/log/itzo", "directory for itzo.log")
 )
 
+func ProcessInstanceParameters() error {
+	klog.V(2).Infof("checking instance parameters")
+	// For now, only AWS is supported. On other platforms we'll fall back to
+	// cloud-init user data.
+	params, err := aws.NewAWSParameters(InstanceParameterBasePath, nil)
+	if err != nil {
+		return err
+	}
+	allParameters, err := params.GetAllParameters()
+	if err != nil {
+		return fmt.Errorf("getting instance parameters: %v", err)
+	}
+	err = os.MkdirAll(ItzoDir, 0755)
+	if err != nil {
+		return fmt.Errorf("ensuring %s exists: %v", ItzoDir, err)
+	}
+	for name, value := range allParameters {
+		pth := filepath.Join(ItzoDir, name)
```

Comment:
REALLY nit: let's name this path :D

Reply:
I usually try not to use package names from the standard library as variable names :) Since we don't import path here, it's fine to rename, though.